Code/GraphMol/DetermineBonds/DetermineBonds.cpp

```diff
@@ -170,8 +171,10 @@ void determineConnectivity(RWMol &mol, bool useHueckel, int charge,
   }
   if (useHueckel) {
     connectivityHueckel(mol, charge);
-  } else {
+  } else if (useVdw) {
     connectivityVdW(mol, covFactor);
+  } else {
+    ConnectTheDots(&mol, ctdIGNORE_H_H_CONTACTS);
```

Comment:
As a hackathon idea, would it be worth extending ConnectTheDots to take optional vdw radii so it could also be the useVdw code path?

Reply:
I'm not sure it's worth it. As Roger pointed out: figuring out whether or not two atoms should be connected ought not to be that tricky (at least as long as we stick to organic molecules)